content/en/docs/external-add-ons/elyra/_index.md

```diff
@@ -1,19 +1,5 @@
 +++
 title = "Elyra"
-description = "Elyra enables data scientists to visually create end-to-end machine learning (ML) workflows."
+description = "Elyra: AI-centric extensions to JupyterLab"
```

Comment:
I would still use the old description here, as in the context of Kubeflow, the most value from Elyra is to help visually create the workflow/pipelines.